packages/react-core/src/layouts/Alignment/Alignment.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { css, getModifier } from '@patternfly/react-styles';
+import styles from '@patternfly/patternfly-next/layouts/Alignment/styles.css';
+import PropTypes from 'prop-types';
+import { componentShape } from '../../internal/componentShape';
+
+const propTypes = {
+  children: PropTypes.any,
+  className: PropTypes.string,
+  component: componentShape,
+  align: PropTypes.string
```

Comment:
Missed this. This should either be PropTypes.oneOf(['left', 'center', 'right']) or similar to.how ButtonVariant is. Maybe `Align`

Reply:
is there any preference for which way to approach this? if I did the route similar to ButtonVariant would it be something like: 
```
export default AlignVariant = {
left: "left",
right: "right".
center: "center
}

const defaultProps = {
align: PropTypes.oneOf(Object.keys(AlignVariant))
}
```